tensorflow/contrib/lite/examples/ios/simple/ios_image_load.h

```diff
@@ -16,10 +16,8 @@
 #define TENSORFLOW_EXAMPLES_IOS_IOS_IMAGE_LOAD_H_
```

Comment:
I would pull the #ifdef guards as they are not standard on Apple platforms ObjC programming, and you have changed this header to no longer just be a C++ header.